src/data/endpoints.ts

```diff
@@ -5,6 +5,7 @@ import { AllLiquidityPools } from "components/SetupPanes/AllLiquidityPools";
 import AllOffers from "components/SetupPanes/AllOffers";
 import AllWithFailed from "components/SetupPanes/AllWithFailed";
 import ClaimableBalances from "components/SetupPanes/ClaimableBalances";
+import Fees from "components/SetupPanes/Fees";
 import FindPaymentPaths from "components/SetupPanes/FindPaymentPaths";
```

Comment:
The SetupPane for "all" was inappropriate for fees. so I created a new one for Fees.

Reply:
I ended up deleting this and returning null. no need to duplicate code for an obvious point.